services/traefik/.env

```diff
@@ -3,4 +3,4 @@ MACHINE_FQDN=osparc.local
 # docker run --rm --entrypoint htpasswd registry:2 -nb user password | sed -e s/\\$/\\$\\$/g
 # echo $(htpasswd -nb user password) | sed -e s/\\$/\\$\\$/g
 TRAEFIK_USER=admin
-TRAEFIK_PASSWORD=$apr1$WOeQr2FL$NAWGz4.pvaV7/YZw.MHJW1
+TRAEFIK_PASSWORD=$apr1$H7Se3sVQ$jXWsAdlwUhxZd54P.AYhX.
```

Comment:
Why commit this password?? Why not create a script that writes down the right one (e.g. in the make pipelines the .env can be ``sed`` and the password replaced )

Reply:
this is a default password (which was wrong before). It is actually replaced using sed already when deploying locally.